### PR TITLE
Fixed bug with csv workspace import, added flashbox in users admin list

### PR DESCRIPTION
**Pull request by ptsavdar**
- Fixed bug in workspace users import with csv
- Added flashbox to users list in admin
- Changed path summary from default "pinned" to default "unpinned"

### CODE REVIEW

File: plugin/path/Resources/public/js/Path/Service/PathService.js [hunk 1]
```diff
@@ -37,7 +37,7 @@
              */
             var summary = {
                 opened: true,
-                pinned: true
+                pinned: false
```
Comment (ngodfraind) [inline plugin/path/Resources/public/js/Path/Service/PathService.js]: This is unrelated to the title of the issue ? Should it be merged aswell ?

Comment (ptsavdar) [inline plugin/path/Resources/public/js/Path/Service/PathService.js]: Yes, I know but to gain some time I pushed it here. It's just a minor detail.

**Comment by ngodfraind** [comment]
LGTM

**Comment by ptsavdar** [comment]
LGTM
